src/lib/receiver.js

```diff
@@ -281,9 +289,13 @@ function createReceiver (opts) {
     }
 
     const fulfillment = cc.toFulfillmentUri(conditionPreimage)
-    debug('about to submit fulfillment: ' + fulfillment)
+
     // returning the promise is only so the result is picked up by the tests' emitAsync
-    return client.fulfillCondition(transfer.id, fulfillment)
+    return reviewPayment(paymentRequest)
```

Comment:
:+1: Fixed so that it rejects to the `reviewPayment`'s error if `reviewPayment` rejects.